### PR TITLE
Add Loading state

### LINKED ISSUE #241: "Register Token" shows "No data" while loading tokens
## Scope

- Currently, there's a bunch of tokens registered on the test network
- When clicking "Register Token", the page however shows a "No Data" message even though the tokens are at that time loaded in the background
- As a user, this is confusing. If there was e.g. a loading screen, I knew that the page was loading in the background
- Currently, I think there's a problem with the chain or bridge-ui

## Deliverables

- Introduce a loading screen for bridge-ui while loading tokens
- [Optional] Paginate tokens when they become too many (maybe a separate ticket)

## Gains for the project

- No more confused users

## Roles
bounty gardener: @TimDaub / 10%
bounty worker: name / 75%
bounty reviewer: name / 15%

### PR DESCRIPTION
**Pull request by iamonuwa**
Fixes #241

### CODE REVIEW

File: src/routes/registerToken.tsx [hunk 1]
```diff
@@ -52,13 +52,17 @@ export default class RegisterToken extends React.Component<RegisterTokenProps> {
...
+        {tokensStore.ready ? (
+          <List
+            itemLayout="vertical"
+            size="small"
+            className="tokens-list"
+            dataSource={tokensStore.ready ? tokensStore.list : undefined}
+            renderItem={item => <Item item={item} />}
+          />
+        ) : (
+          <Spin />
```
Comment (MaxStalker) [inline src/routes/registerToken.tsx]: Personally, I would add some message here. 
Something like "Loading a list of tokens, please wait..."

File: src/routes/registerToken.tsx [hunk 2]
```diff
@@ -52,13 +52,20 @@ export default class RegisterToken extends React.Component<RegisterTokenProps> {
...
+        {tokensStore.ready ? (
+          <List
+            itemLayout="vertical"
+            size="small"
+            className="tokens-list"
+            dataSource={tokensStore.ready ? tokensStore.list : undefined}
+            renderItem={item => <Item item={item} />}
+          />
+        ) : (
+          <div className="tokens-loading">
```
Comment (MaxStalker) [inline src/routes/registerToken.tsx]: I would recommend to at least swap `span` and `Spin` component. In best case - to make it look like [here](https://ant.design/components/spin/)

Comment (iamonuwa) [reply src/routes/registerToken.tsx]: Alright, I'll fix this

Comment (iamonuwa) [reply src/routes/registerToken.tsx]: I've checked the link, not sure what exactly you are looking out for here. I implemented the Spin

Comment (MaxStalker) [reply src/routes/registerToken.tsx]: image
When I said swap `span` and `Spin` I've meant to make it look like this 👆

**Pull request merged**